srcpkgs/yadm/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'yadm'
 pkgname=yadm
-version=3.0.2
+version=3.1.0
 revision=1
 depends="bash git gnupg"
 short_desc="Yet Another Dotfiles Manager"
 maintainer="John <me@johnnynator.dev>"
 license="GPL-3.0-or-later"
 homepage="https://yadm.io/"
 distfiles="https://github.com/TheLocehiliosan/${pkgname}/archive/${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/TheLocehiliosan/yadm/archive/${version}.tar.gz"
```

Reply:
Does it make any difference? There's also `vbin ${pkgname}` and `vman ${pkgname}.1`. But don't fix what's not broken...

Reply:
This one is for convenience, allows easy copy paste to go to their github repo.

Reply:
I rather the maintainer decide on that. I just want tab completion to work.

I guess this could be decided in general as part of #27735.